### ISSUE TITLE
Getting "Segmentation fault" when importing a module

### ISSUE THREAD

**Issue by Yass1r** [HavocFramework/Havoc#96]
Hi,
I would like to report that the client crashes when trying to import a module using scripts manager. 
Thanks

**Comment by Cracked5pider** [comment]
Can you retest it? Pushed an update. and let me know more on how it crashes.

**Comment by Yass1r** [comment]
It worked .. Thanks for fixing this issue. I notice also when powerpick executes a PowerShell command such as `powerpick whoami` it executes the PowerShell in non-hidden windows. That means if you watch the Windows desktop, it will show a blue screen very quick and disappear!
PowerPick doesn't work on Windows server 12 .. :)

**Comment by Yass1r** [comment]
It is still unstable! I'm getting crashed again. I am importing the python script using Script manager, and then getting segmentation fault on the terminal

**Comment by Cracked5pider** [comment]
What OS are you? And what script did you load?

**Comment by Yass1r** [comment]
The latest version of Kali ... Trying both. Currently, both are not loading for me! I have to test it on another VM and will let you know ..

**Comment by mgeeky** [comment]
Can confirm. Client crashes for me too on Kali, as soon as I load up `powerpick.py`:

```
Linux kali 5.18.0-kali7-amd64 #1 SMP PREEMPT_DYNAMIC Debian 5.18.16-1kali1 (2022-08-31) x86_64 GNU/Linux
```

We could use a little more verbose exception output whenever client/teamserver crashes to help you out nail down any future issues :-)

**Comment by Cracked5pider** [comment]
Alright, I am going to check it out thanks for reporting. Haven't tested it very often on Kali.

**Comment by Cracked5pider** [comment]
this issue should be resolved in commit: [3e49c03](https://github.com/HavocFramework/Havoc/commit/3e49c03a86069a4ce124b74e2525477bcdb8c960)
let me know if it's working now. If it does then close the issue.

**Comment by chbGSmCm** [comment]
Please reopen if the issue persists.
